### ISSUE TITLE
Error in documentation on CSP frame-ancestors configuration

### ISSUE THREAD

**Issue by subjugum** [vector-im/element-web#21095]
### Steps to reproduce

Configure nginx with the headers from https://github.com/vector-im/element-web/#configuration-best-practices
Then try to open a Jitsi widget in a room.

#### What did you expect?
Jitsi widget loading.

#### What happened instead?
Jitsi widget not loading and failing with a CSP error.
```
Content Security Policy: The page’s settings blocked the loading of a resource at https://element.example.com/ (“frame-ancestors”).
```

#### How did you fix this?
Docs tell you to set this:
```
add_header X-Frame-Options SAMEORIGIN;
add_header Content-Security-Policy "frame-ancestors 'none'";
```
I set `frame-ancestors 'self'` and the widget starts loading.

### Will you send logs?

No

**Comment by dbkr** [comment]
Were you also hosting the jitsi meet web with the same headers? They should apply to parent frames, not children, so the intention would be for element to be served with those headers but the jitsi webapp to allow embedding, which I think should work?

**Comment by subjugum** [comment]
To be clear: The Jitsi widget not loading happens before a request to the embedded Jitsi itself is ever made. Visually the widget keeps loading and not even getting to the "Join Conference" button. Only after pressing that button the requests to Jitsi Meet happen. For completeness sake: I tested this with jitsi.riot.im and meet.element.io.

The thing that should trip you up this part in your documentation:
```
add_header X-Frame-Options SAMEORIGIN;
add_header Content-Security-Policy "frame-ancestors 'none'";
```
Both your docs and [the thing you're linking to](https://cheatsheetseries.owasp.org/cheatsheets/Clickjacking_Defense_Cheat_Sheet.html) are saying that frame-ancestors is just a modern version of X-Frame-Options. `X-Frame-Options SAMEORIGIN` equals `frame-ancestors 'self'`, which is exactly what I changed to make it work.
